tests/test_mysql_full_and_incremental.py

```diff
@@ -0,0 +1,675 @@
+                                        'tap_stream_id': 'tap_tester_mysql_0-simple_example'},
+                     'various_types': {'annotated_schema': None,
+                                       'is_view': False,
+                                       # 'key_properties': ['c_pk'],
+                                       'key_properties': [],
+                                       'row_count': len(dummy_data['various_types']),
+                                       'schema_name': None,
+                                       'selected': False,
+                                       'stream': 'various_types',
+                                       'stream_alias': None,
+                                       'stream_id': 17560,
+                                       'stream_name': 'various_types',
+                                       'table_name': 'various_types',
+                                       'tap_stream_id': 'tap_tester_mysql_0-various_types'},
+                     'incremental': {'annotated_schema': None,
+                                     'schema_name': None,
+                                     'stream': 'incremental',
+                                     'stream_alias': None,
+                                     'stream_id': 19254,
+                                     'stream_name': 'incremental',
+                                     'table_name': 'incremental',
+                                     'tap_stream_id': 'tap_tester_mysql_0-incremental'},
+                     'my_isam': {'annotated_schema': None,
+                                 'database_name': 'tap_tester_mysql_1',
+                                 'is_view': False,
+                                 'key_properties': [],
+                                 # 'key_properties': ['c_pk'],
+                                 'replication_key': None,
+                                 'replication_method': None,
+                                 'row_count': 1,
+                                 'schema_name': None,
+                                 'selected': False,
+                                 'stream': 'my_isam',
+                                 'stream_alias': None,
+                                 'stream_id': 20716,
+                                 'stream_name': 'my_isam',
+                                 'table_name': 'my_isam',
+                                 'tap_stream_id': 'tap_tester_mysql_1-my_isam'},
+                     'view': {'annotated_schema': None,
+                              'schema_name': None,
+                              'stream': 'view',
+                              'stream_alias': None,
+                              'stream_id': 17559,
+                              'stream_name': 'view',
+                              'table_name': 'view',
+                              'tap_stream_id': 'tap_tester_mysql_1-view'}}
+
+class MySQLFullAndIncremental(unittest.TestCase):
+    def setUp(self):
```

Comment:
Is there any setup code which could be shared and placed into a base test class?

Reply:
Likely, seems like it might be more effort than worth it right now.